clusterloader2/testing/load/config.yaml

```diff
@@ -526,6 +527,7 @@ steps:
     Method: PodStartupLatency
     Params:
       action: gather
+      enableViolations: true
```

Comment:
I don't think PodStartupLatency has enableViolations field: https://github.com/kubernetes/perf-tests/blob/c8098dfa0d3c09becd3f669b9ba2b49e413c5bda/clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go#L73-L95

Please remove

Reply:
ah true, I though it uses prometheus metric...